src/test/java/fi/aalto/cs/apluscourses/model/ModuleInstallerTest.java

```diff
@@ -0,0 +1,294 @@
+package fi.aalto.cs.apluscourses.model;
```

Comment:
This test gets stuck for me locally (which I suspect is also why travis fails)

Reply:
Hopefully this is now fixed.